docs/overview.rst

```diff
@@ -44,6 +44,8 @@ These computers should all physically be in your organization's office.
 
 .. _`Tails operating system`: https://tails.boum.org
 
+.. _securedrop_architecture:
```

Comment:
Minor: `securedrop_architecture` is a little vague. `securedrop_architecture_diagram` would be better?